prow/cmd/branchprotector/protect.go

```diff
@@ -188,51 +191,70 @@ func (p *Protector) ConfigureBranches() {
 func (p *Protector) Protect() {
 	bp := p.cfg.BranchProtection
 
-	// Scan the branch-protection configuration
-	for orgName, org := range bp.Orgs {
-		if err := p.UpdateOrg(orgName, org, bp.Protect, bp.Contexts, bp.Pushers); err != nil {
-			p.errors.add(err)
-		}
-	}
+	// TODO(fejta): check for deprecated usage and warn
+	policy := bp.MakePolicy()
 
-	// Do not automatically protect tested repositories
-	if !bp.ProtectTested {
-		return
-	}
+	if bp.ProtectTested { // Automatically require any required prow jobs for unconfigured repos
```

Comment:
Shouldn't we also require `ProwJobs` for repos that happen to have something already configured?